doc/cookbook/source/examples/base_api/dense_dispatching.rst

```diff
@@ -0,0 +1,25 @@
+==================================================
+Dense Dispatching for Linear Discriminant Analysis
+==================================================
+
+Some algorithms like Linear Discriminant Analysis, Least Angle Regression, etc. in shogun support training with features of multiple primitive type.
```

Comment:
Be more general.
"Dense features, like those read from a sgclass(CCSVFIle), can be read into memory in multiple primitive types, e.g. 32 bit or 64 bit floating point numbers.
Downstream algorithms can often deal with multiple of such feature types, and will carry out required computations in the provided primitive type.
Reducing from 64 bit float to 32 bit float can be beneficial if for example very large matrices have to be stored.